apps/web/content/docs/typography/list.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: React Lists - Flowbite
-description: Use the list component to show an unordered or ordered list of items based on multiple styles, layouts, and variants built with Tailwind CSS and Flowbite
+description: Use the list component to show an unordered or ordered list of items based on multiple styles, layouts, and variants built with Tailwind CSS and Flowbite.
```

Comment:
no need for `.` at the end

Reply:
the coderabbitai mentioned that should be there!!
[https://github.com/themesberg/flowbite-react/pull/1337/files/ab4911d15d459234d2596321fa5241b14fc8c560#diff-8d208e84b68cdceb89656de56d5cbb181bb77329e8c365e7d753c8cfd8a98084](https://github.com/themesberg/flowbite-react/pull/1337/files/ab4911d15d459234d2596321fa5241b14fc8c560#diff-8d208e84b68cdceb89656de56d5cbb181bb77329e8c365e7d753c8cfd8a98084)

Reply:
never trust an AI 😄